datafusion/sqllogictest/test_files/unnest.slt

```diff
@@ -33,8 +33,8 @@ AS VALUES
 statement ok
 CREATE TABLE nested_unnest_table
 AS VALUES 
-    (struct('a', 'b', struct('c')), (struct('a', 'b', [10,20])), [struct('a', 'b')]), 
-    (struct('d', 'e', struct('f')), (struct('x', 'y', [30,40, 50])), null)
+    (struct(['a'], 'b', struct('c')), (struct('a', 'b', [[10,20]])), [struct(['a'], 'b')]), 
```

Comment:
would it be possible to please leave the original test the same (so we can verify the existing behavior is unchanged) and add new tests for the newly added features?

Reply:
i revert the change on this table and created a new one